### DISCUSSION TITLE
Unable to enable input control on lenovo xiaoxin pad 2024

### DISCUSSION THREAD

**Discussion by tempaone** [rustdesk/rustdesk#9462] [General]
### Bug Description

I cant enable input control feature on this tablet because i believe the restricted settings is blocking it, but there is no option to allow restricted settings.

![Screenshot_20240925-230429](https://github.com/user-attachments/assets/ab30967b-f305-424d-9e68-b789bfd27f3b)
![Screenshot_20240925-230639](https://github.com/user-attachments/assets/87974e97-da18-402a-9dfa-c97039e27922)
![Screenshot_20240925-230649](https://github.com/user-attachments/assets/b6c17c1e-19bf-4cd6-812b-4bd654d037c0)

### How to Reproduce

Try on this particular tablet.

### Expected Behavior

enabling restricted settings in the app settings via top right dots, or somewhere in the app information to enable restricted settings.

### Operating system(s) on local side and remote side

andriod 14,  zui 16.0.447

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

![Screenshot_20240925-230409](https://github.com/user-attachments/assets/d2efc1d4-1b6c-4f9f-add6-1e7094b5dd2c)
![Screenshot_20240925-230416](https://github.com/user-attachments/assets/6d2ecfdc-b997-42ed-9016-eceeec20a47a)

**Comment by lukas-runge** [comment]
Installing RustDesk via SAI fixes this: https://play.google.com/store/apps/details?id=com.mtv.sai
